source/migrating_from_legacy_products/index.html.md.erb

```diff
@@ -0,0 +1,137 @@
+---
```

Comment:
Any chance we can rename the folder to kebab case?

Reply:
This is clearly a weakness of mine! Thanks for spotting - fixed now.